pkgs/development/tools/flatpak-builder/default.nix

```diff
@@ -32,7 +32,7 @@
 , gnutar
 , json-glib
 , libcap
-, libdwarf
+, libdwarf_0_4
```

Comment:
`libdwarf` is not even used any more: https://github.com/flatpak/flatpak-builder/commit/ee8b921e9169e3f46a5e0e62b2bc583ecebdfe96

Reply:
Fixed, thanks!